### ISSUE TITLE
2b.  Images belong to individual profiles

### ISSUE THREAD

**Issue by Moirvae** [cocaseattle/artist-directory#25]
In the ProfileGrid plugin, photos appear to belong to groups as a whole, not individual profiles.

**Comment by HTD-CoCA** [comment]
Working on having Profile Grid's premium feature able to fix this issue.  Will check if the feature has been downloaded and activated.

**Comment by HTD-CoCA** [comment]
Issue resolved.  Using this feature would involve a user/artist clicking the +New button at the top left corner and select new post to get to the Block Editor.  Closing issue.

**Comment by Moirvae** [comment]
Still not seeing this. I see photos on a group level, not an individual level.

**Comment by HTD-CoCA** [comment]
You need to go to blogs and under the blog section they can add A gallery
using Gutenberg block Editor or NextGen Gallery Gutenberg gallery also in
the block editor.

**Comment by Moirvae** [comment]
Yes, I saw the editor & gallery plugins. How are they intended to work with/interact with the profile plugin?

**Comment by HTD-CoCA** [comment]
They are not intended to work with/interact with the profile grid plugin.  Archis was going to compose instructions with screenshots of how the Artists can use Guttenberg & possibly NextGen Gallery.
